invenio_records_resources/records/api.py

```diff
@@ -100,6 +100,12 @@ def record(self):
         """Get the file's record."""
         return self.record_cls(self._record.data, model=self._record)
 
+    @classmethod
+    def remove_many(cls, record_file_ids):
+        """Hard delete record file instances."""
+        record_files = cls.model_cls.query.filter(cls.model_cls.id.in_(record_file_ids))
```

Comment:
# for reviewers (important): 
I would like to get your feedback on this, it is a bulk removal so quite delicate operation, equivalent of previously used https://github.com/inveniosoftware/invenio-records/blob/master/invenio_records/api.py#L506 (we used this method for removing file record instances one by one.
the problem is that in that method we have also implemented signals (pre and post delete), do you see any possible consequences of not having signals for file record? I might have not thought about everything

Reply:
We basically "deprecated" record signals for InvenioRDM, so I don't see any issues.